### PR TITLE
Cis 5.2.7

### PR DESCRIPTION
**Pull request by lsm5**
#### Description:

- Add template to scc_limit_net_raw_capability to check if NET_RAW or ALL are dropped in at least one SCC

#### Rationale:

- Critical to drop dangerous capabilities

NOTE: We aren't yet compliant with this.

@redhatrises @jhrozek @yuumasato PTAL

### CODE REVIEW

File: applications/openshift/scc/scc_limit_net_raw_capability/rule.yml [hunk 1]
```diff
@@ -27,7 +27,26 @@ ocil_clause: 'NET_RAW does not exist or is configured in defaultAddCapabilities
...
+    vars:
+        ocp_data: "true"
+        filepath: /apis/security.openshift.io/v1/securitycontextconstraints
+        yamlpath: ".items[:]['requiredDropCapabilities']"
+        check_existence: "at_least_one_exists"
+        entity_check: "at least one"
+        values:
+          - type: "string"
+            value: "NET_RAW"
+            value: "ALL"
```
Comment (lsm5) [inline applications/openshift/scc/scc_limit_net_raw_capability/rule.yml:52]: hmm, @redhat-rmcallis @jhrozek so IIUC, 5.2.9 needs "ALL". Is it ok for ALL to be present here as well? The CIS draft does mention NET_RAW or ALL for 5.2.7

File: applications/openshift/scc/scc_limit_net_raw_capability/rule.yml [hunk 2]
```diff
@@ -27,7 +27,26 @@ ocil_clause: 'NET_RAW does not exist or is configured in defaultAddCapabilities
...
+    - general: |-
+        {{{ openshift_cluster_setting("/apis/security.openshift.io/v1/securitycontextconstraints") | indent(8) }}}
+{{%- endif %}}
+
+template:
+    name: yamlfile_value
+    vars:
+        ocp_data: "true"
+        filepath: /apis/security.openshift.io/v1/securitycontextconstraints
+        yamlpath: ".items[:]['requiredDropCapabilities']"
```
Comment (jhrozek) [inline applications/openshift/scc/scc_limit_net_raw_capability/rule.yml:46]: I think we have the same issue as in PR #6250 with the fact that `requiredDropCapabilities` is an array...

**Comment by jhrozek** [comment]
I rebased the patch atop origin/master and added fixups that were both possible and needed since PR#6278

**Comment by lsm5** [comment]
@jhrozek ack thanks. Sorry I haven't yet updated this. The other PRs are on my TODO btw.

**Comment by jhrozek** [comment]
> @jhrozek ack thanks. Sorry I haven't yet updated this. The other PRs are on my TODO btw.

Oops, sorry, I was testing the changes that were needed to even enable the tests in this PR and just ended up fixing both of the 5.2 open ones. I'll let you fixup the rest (no rush though!)

**Comment by jhrozek** [comment]
Merging since Lokesh is OK with the fixups.

**Pull request merged**